File: PHP/DisemVowelTrolls/MySolution.php
<?php
  
function disemvowel(string $s): string  {
  $vowels = array("a", "e", "i", "o", "u", "A", "E", "I", "O", "U");

  return str_replace($vowels, "", $s);
}

// Original kata: https://www.codewars.com/kata/52fba66badcd10859f00097e
// My solution: https://www.codewars.com/kata/reviews/63947bd335232800016cca64/groups/6735f3d71c250d9417d76b83

File: Rust/CountByX/MySolution.rs
fn count_by(x: u32, n: u32) -> Vec<u32> {
    let mut v = vec![];
    for i in (x..= x * n).step_by(x as usize) {
        v.push(i)
    }
    v
}

// Original kata: https://www.codewars.com/kata/5513795bd3fafb56c200049e
// My solution: https://www.codewars.com/kata/reviews/62efe23a7814d50001a68fc0/groups/67e3ac10ee16b8129e3f70d3
